src/components/Fetch/index.js

```diff
@@ -527,7 +532,8 @@ class Fetch extends React.Component {
                 <FormGroup controlId="findInput" className="filter-header">
                   <Col lg={6} ><FormControl type="text"
                     placeholder="optional. regexp to search for"
-                    inputRef={ref => { this.findInput = ref; }}/></Col>
+                    inputRef={ref => { this.findInput = ref; }}
+                    onChange={this.handleChangeFind.bind(this, this.state.caseSensitive)}/></Col>
```

Comment:
See https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md. Long term, I want to turn more warnings on.

Also i think the bind is unnecessary, so you can probably reduce it to `this.find()`